app/javascript/chat/actions/requestActions.js

```diff
@@ -92,3 +92,52 @@ export async function updateMembership(membershipId, userAction) {
 
   return response.json();
 }
+
+/**
+ *
+ * @param {string} feedback_message
+ * @param {string}  type_of_feedback
```

Comment:
**nitpick (non-blocking):** There seems to be an extra space before this param name and the names not being properly aligned massively triggers my OCD 😉 

```suggestion
 * @param {string} type_of_feedback
```